### PR TITLE
Story/10125 recycle bin show deletion date

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
- Adding deletion date on items in recycle bin
- Change in text
- Change in font colour (as per figma) 
- Changes to unit tests

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10125

## Screenshots
WhatsApp Image 2022-01-13 at 14 24 53

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Comment by naveeddotio** [comment]
> When I compare the screenshot with Figma it seems like the distance between new text and the old info is a little to big.

@marcussc That is fixed already in the code.

**Comment by naveeddotio** [comment]
> When I compare the screenshot with Figma it seems like the distance between new text and the old info is a little to big.

@marcussc Fixed and updated the screenshot.

File: src/xcode/ENA/ENA/Source/Scenes/RecycleBin/Cells/RecycleBinItemCellModel.swift [hunk 1]
```diff
@@ -123,6 +123,18 @@ class RecycleBinItemCellModel {
 		}
 	}()
 
+	lazy var quaternaryInfo: String? = {
+		guard let expirationDate = Calendar.current.date(byAdding: .day, value: 30, to: recycleBinItem.recycledAt) else {
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/RecycleBin/Cells/RecycleBinItemCellModel.swift:127]: Could we make the `30` a static let on `RecycleBin` or something like that so we have a single place to change that value if we ever decide to change it?

**Review by nickguendling — CHANGES_REQUESTED** [review]

**Review by marcussc — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Pull request merged**